plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -782,6 +782,21 @@ class GesturesPluginTest {
   fun obtainMotionEventActionLater(action: Int): MotionEvent {
     return MotionEvent.obtain(200, 500, action, 15.0f, 10.0f, 0)
   }
+
+  @Test
+  fun verifyDefaultShoveGestureAngle() {
+    verify {
+      gesturesManager.shoveGestureDetector.maxShoveAngle = GesturesPluginImpl.MAX_SHOVE_ANGLE
```

Comment:
do you want to check values or only that you set ANY value?
better to make `MAX_SHOVE_ANGLE` from GesturesPluginImpl private in companion object
and add same comp object to this test class

```
gesturesManager.shoveGestureDetector.maxShoveAngle = MAX_SHOVE_ANGLE // from test class obj
```

because when will you change any of this values you will not detect such changes in tests.

Reply:
even you can change type of this value and tests will not detect it

Reply:
this test is to validate the default angle set in GesturePluginImpl class. that should be a constant. Although, I get your point that this can also have different values (set programatically). That should be handled in different test imo. I can probably add another test to check that condition too.

Reply:
As for me, default value is a business logic. You can try to change it in comp object and tests will passed

Reply:
Yeah, true. made the changes 👍